archive/s/swift/reverse-string.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+
+guard CommandLine.arguments.count > 0 else {
+    exit(0)
+}
+
+let usersString = CommandLine.arguments[0]
```

Comment:
Just tried testing this in the [online Swift playground](https://iswift.org/playground). It seems `CommandLine.arguments[0]` points to the file name (granted I can't seem to pass args to the script anyway). Is this an issue?

Reply:
Note that Swift's Strings use Unicode, so they have inherent support for emoji. 

Also, Swift isn't as commonly used on the command line right now, which might be why these online editors have such a poor track record with handling command line arguments

Reply:
Makes sense! I anticipated running into these kinds of issues with all these different languages.